res/lang/news/news_pt-br.txt

```diff
@@ -60,30 +60,30 @@ BOMB_DETONATION_IN_TVTOWER_FOUND_CLUES_TEXT  = Autoridades investigaram o atenta
 NEWS_ANNOUNCE_MOVIE_ACTOR_IS_DIRECTOR_TITLE       = Novo filme dirigido e estrelado por %DIRECTORNAME1%
 NEWS_ANNOUNCE_MOVIE_ACTOR_IS_DIRECTOR_DESCRIPTION = Diretor e ator principal %DIRECTORNAME1% tem um novo filme «%MOVIETITLE%» na produção.
 
-#NEWS_ANNOUNCE_MOVIE_NO_CAST_TITLE       = 
-#NEWS_ANNOUNCE_MOVIE_NO_CAST_DESCRIPTION = 
+NEWS_ANNOUNCE_MOVIE_NO_CAST_TITLE       = Em breve no cinema: «%MOVIETITLE%»
+NEWS_ANNOUNCE_MOVIE_NO_CAST_DESCRIPTION = Publicado o primeiro trailer de «%MOVIETITLE%».
 
-#NEWS_ANNOUNCE_MOVIE_NO_DIRECTOR_TITLE       = 
-#NEWS_ANNOUNCE_MOVIE_NO_DIRECTOR_DESCRIPTION = 
+NEWS_ANNOUNCE_MOVIE_NO_DIRECTOR_TITLE       = Em breve no cinema: «%MOVIETITLE%»
```

Comment:
the quotation marks differ to the one in 
`TOASTMESSAGE_BOMB_LICENCE_DESTROYED_TEXT     = A explosão destruiu '%TITLE%'.`